pages/common/lex.md

```diff
@@ -2,6 +2,7 @@
 
 > Lexical analyser generator.
 > Given the specification for a lexical analyser, generates C code implementing it.
+> More information: <https://manned.org/lex.1>.
```

Comment:
`lex.1` refers to the command `flex`. Is this on purpose?

Reply:
Yes, all `lex` in use nowadays is actually `fast lex`, also known as `flex`.